assets/scss/_base.scss

```
@@ -65,8 +65,8 @@ footer {
   .button {
     display: inline-block;
     border-radius: 6px;
-    padding: 0 20px;
-    line-height: 40px;
+    padding: 8px 20px;
```

```suggestion
    padding: 6px 20px;
```

Again, I think you could go smaller :)